R/input-file.R

```diff
@@ -120,7 +121,7 @@ fileInput <- function(inputId, label, multiple = FALSE, accept = NULL,
     tags$div(
       id=paste(inputId, "_progress", sep=""),
       class="progress progress-striped active shiny-file-input-progress",
-      tags$div(class="progress-bar")
+      tags$div(class="progress-bar progress-bar-striped")
```

Comment:
Consider just getting rid of the stripes. They look a bit dated.

Reply:
Make sure that we don't need to support animated stripes (i.e. indeterminate state)

Reply:
I'm not sure this is a great idea for the `Progress` API...it seems like there should be some visual cue for indeterminate state

Reply:
We actually don't have indeterminate state for the Progress API. We just hide the progress bar if we don't have a number.

Reply:
Maybe I don't understand what you mean by indeterminate state...I was thinking of a long running computation, like:

```r
progress <- Progress$new(session, min=1, max=3)
progress$set(message = 'Calculation in progress', detail = 'This may take a while...')
progress$set(value = 2)
Sys.sleep(5)
progress$set(value = 3)
progress$close()
```